tg-portfolio/components/about/Services.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Services = () => {
+  const services = [
+    {
+      id: 'service-1',
+      title: 'Full Stack Development',
+      image: '/icons8-stack-64.png',
+    },
+    {
+      id: 'service-2',
+      title: 'Web Development',
+      image: '/webDev.png',
+    },
+    {
+      id: 'service-3',
+      title: 'Freelance',
+      image: '/freelance.png',
+    },
+    {
+      id: 'service-4',
+      title: 'Consulting',
+      image: '/consulting1.png',
+    },
+  ];
+
+  return (
+    <div className="mt-10 md:mx-3 lg:mx-24 sm:mx-0">
+      <div className="md:p-6 sm:p-2 rounded-lg bg-neutral/60 animate-in slide-in-from-bottom-48 duration-1000">
+        <h1 className="test text-4xl text-black font-bold border-green-700/100 pb-8 px-8 border-b-4">Services</h1>
+        <div className="flex flex-auto justify-between sm:gap-5 flex-row p-5 overflow-auto scrollbar-hide">
+          {services.map((service) => (
+            service.id !== 'service-4' ? (
```

Comment:
Instead of checking the condition this way, you can implement a more efficient check:

We don't want to use | | for the last element. For that, we can get the length of the services array. And map function provides us with an index property along with the item i..e

```javascript
map((service, index))
```

You can use these two things and check whether the current index of the item equals the length of the array. 
If it does, hide the | | div. Else not. 

Hint: Use tailwind CSS property on | | div. Property name is display. According to your condition, you can change the value of the display 

It'll be a dynamic check.